frame/bridge/ethereum/relay/src/helper.rs

```diff
@@ -0,0 +1,43 @@
+// This file is part of merkle-mountain-range from nervosnetwork
+fn log2(mut n: u64) -> u64 {
+	let mut k = 0;
+	while n > 1 {
+		k += 1;
+		n >>= 1;
+	}
+	k
+}
+
+pub fn leaf_index_to_pos(index: u64) -> u64 {
```

Comment:
I've made a PR to ckb_merkle_mountain_range:

https://github.com/nervosnetwork/merkle-mountain-range/pull/9/files

Should use this merged version for performance concern and code reuse.

Reply:
Thanks for remind.